ethers-solc/src/lib.rs

```diff
@@ -426,6 +426,36 @@ impl<T: ArtifactOutput> Project<T> {
     pub fn flatten(&self, target: &Path) -> Result<String> {
         self.paths.flatten(target)
     }
+
+    /// Returns standard-json-input to compile the target contract
+    pub fn standard_json_input(&self, target: &Path) -> Result<String> {
```

Comment:
I'd rather return a the `CompilerInput` here instead

Reply:
@mattsse I make it return string consistent with `flatten`. wdyt?

https://github.com/gakonst/foundry/blob/908c0c8ac47d9ff4ab0494a00fc2896be410e47b/cli/src/cmd/forge/verify.rs#L103-L123

Reply:
still, I think `CompilerInput` would make more sense here and could be useful to for example use it directly with `Solc::compile`

Reply:
ok let me update it

Reply:
@mattsse updated 👍